README.md

```diff
@@ -43,10 +43,11 @@ Logging of this plugin has several categories:
 * `frontendUiTable`
 * `gitCore`
 
-By default, IntelliJ logs everything with level `INFO` and above into `idea.log` file (localization depends on IntelliJ version and OS). To find where this file is located, go to `Help` -> `Show Log in Files`.
+By default, IntelliJ logs everything with level `INFO` and above into `idea.log` file.
+The exact location depends on IntelliJ version and OS, check `Help` -> `Show Log in Files` to find out.
 
 To enable logging in `DEBUG` level, add selected categories to list in `Help` -> `Diagnostic Tools` -> `Debug Log Settings`.
-For our classes, the categories are customized and provided above; by default, the category is a fully-qualified name of each class.
+Typically with Java logging, the category is a fully-qualified name of each class; in our case, however, we're only ever using the categories provided above.
```

Comment:
Some IDE language tool (I don't know where did it get from :rofl: ) suggests 3 "improvements" in our readme. Please, consider:

Line 28: 
- `s/for IDE/for the IDE/`
- `s/2048MB/2048 MB/`

Line 29:
- comma after `Typically`

Reply:
Done on PR #263